### PR TITLE
Adding state names to results

### PR DESCRIPTION
**Pull request by albrja**
### Translates census state ids to state names in final outputs.
- *Category*: Post-processing
- *JIRA issue*: [MIC-3760](https://jira.ihme.washington.edu/browse/MIC-3760)
- *Research reference*: N/A

### Changes and notes
-Formats state column in formatters for post-processing
-Updates load data for post-processing to handle columns not present in all observers.

### Verification and Testing
Successfully ran make results.

### CODE REVIEW

File: src/vivarium_census_prl_synth_pop/results_processing/formatter.py [hunk 1]
```diff
@@ -23,13 +25,19 @@ def format_simulant_id(data: pd.DataFrame) -> pd.Series:
     return data["random_seed"].astype(str) + "_" + data["simulant_id"].astype(str)
 
 
+def get_state_name(data: pd.DataFrame, metadate=None) -> pd.Series:
```
Comment (mattkappel) [inline src/vivarium_census_prl_synth_pop/results_processing/formatter.py:28]: typo: s/metadate/metadata

Comment (mattkappel) [reply src/vivarium_census_prl_synth_pop/results_processing/formatter.py:28]: Or wait, what is `metadate` supposed to be?

Comment (albrja) [reply src/vivarium_census_prl_synth_pop/results_processing/formatter.py:28]: Oops don't know how that got there.  Not even an arg that is needed.

File: src/vivarium_census_prl_synth_pop/results_processing/formatter.py [hunk 2]
```diff
@@ -23,13 +25,19 @@ def format_simulant_id(data: pd.DataFrame) -> pd.Series:
     return data["random_seed"].astype(str) + "_" + data["simulant_id"].astype(str)
 
 
+def get_state_name(data: pd.DataFrame) -> pd.Series:
+    state_map = dict((v, k) for k, v in metadata.CENSUS_STATE_IDS.items())
```
Comment (rmudambi) [inline src/vivarium_census_prl_synth_pop/results_processing/formatter.py:29]: This is equivalent to a dict comprehension, no?

**Review by rmudambi — APPROVED** [review]

**Pull request merged**